src/introduction/introduction.component.js

```diff
@@ -29,49 +34,41 @@ export default function Introduction(props) {
           applications.
         </section>
         <section>
-          The playground isn't the only way to use single-spa, but is one of the
-          easier ways. It will set you up with single-spa,{" "}
-          <a href="https://github.com/systemjs/systemjs">systemjs@6</a>, and{" "}
-          <a href="https://github.com/WICG/import-maps">import maps</a>. This
-          lets you do the following:
+          The playground will guide you to build a successful singe-spa
+          application, highlighting important steps and give tools to help you
+          understand what went wrong. Mind that this isn't the only way to use
+          single-spa, but by better understanding the concepts, you'll be able
+          to apply it in any way you like. Here's what we'll do:
           <ol>
-            <li>Have microfrontends that are in separate git repos.</li>
             <li>
-              Use different frameworks in the microfrontends, if you wish.
+              Have micro-frontends that are in separate git repos. We will be
+              focusing on React, but you could use Angular, Vue, etc
             </li>
-            <li>Deploy your microfrontends independently.</li>
+            <li>Test if your microfrontend is build up correctly</li>
             <li>
-              Not have to worry about some of the initial setup and
-              configuration.
+              Understand and create a{" "}
+              <a href="https://single-spa.js.org/docs/configuration">
+                root config
+              </a>
+              , which will do the routing of your micro-frontends
             </li>
+            <li>Deploy your micro-frontends independently.(coming soon)</li>
```

Comment:
Are you planning on working on this within the playground?

Perhaps for now we could just link to [these docs](https://single-spa.js.org/docs/recommended-setup#deployment-and-continuous-integration-ci)?

Reply:
Since it's one of the upsides of single-spa and microfrontends, helping others successfully deploy their apps seems to me like a valid next step! But not for now, this PR is so big that it can't fit anything else! sorry for making you read them top to bottom every time I make changes 😔

Reply:
I agree - that would be a great thing to add to the playground as a next step